### PR TITLE
🔨♻️Profiling tools for catalog

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

- ⚠️ `docker-compose`: adds prefix with initials of service name to `hostname`
   - `POSTGRES_CLIENT_NAME` takes this value
   - Facilitates identifying db connections in the `processes lists`  of `postgres`
- ⚗️ increases RPC client timeout for new services calls (experimental) @odeimaiz 
- 🔨 Adds profiler in catalog's RPC

## Related issue/s

- Fixes on #5964 

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by matusdrobuliak66 — APPROVED** [review]
:+1:

**Review by odeimaiz — APPROVED** [review]
thanks

**Review by YuryHrytsuk — APPROVED** [review]
Very nice! Thank you!

File: services/docker-compose.yml [hunk 1]
```diff
@@ -390,7 +390,7 @@ services:
   invitations:
     image: ${DOCKER_REGISTRY:-itisfoundation}/invitations:${DOCKER_IMAGE_TAG:-latest}
     init: true
-    hostname: "{{.Node.Hostname}}-{{.Task.Slot}}"
+    hostname: "inv-{{.Node.Hostname}}-{{.Task.Slot}}"
```
Comment (sanderegg) [inline services/docker-compose.yml:393]: ohlala... is this for the RabbitMQ queue names? maybe we should add the application name in the client side to alleviate this stuff.

**Review by sanderegg — APPROVED** [review]

**Review by GitHK — APPROVED** [review]

File: services/docker-compose.yml (hunk 1 above)
Comment (pcrespov) [reply services/docker-compose.yml:393]: i recall discussing with @YuryHrytsuk about the limits of HOSTNAME. That is why i just add a small prefix

File: packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/catalog/services.py [hunk 2]
```diff
@@ -52,6 +53,7 @@ async def _call(
             user_id=user_id,
             limit=limit,
             offset=offset,
+            timeout_s=2 * RPC_REQUEST_DEFAULT_TIMEOUT_S,
```
Comment (mrnicegyu11) [inline packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/catalog/services.py:56]: Magic number multipllier `2` hidden in code detected ⚠️

**Review by mrnicegyu11 — APPROVED** [review]
lgtm, minor comment ;)

File: packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/catalog/services.py (hunk 2 above)
Comment (pcrespov) [reply packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/catalog/services.py:56]: thx. this is under control :-)

**Pull request merged**